runtime/kusama/src/lib.rs

```diff
@@ -1267,7 +1267,7 @@ pub type XcmRouter = (
 
 parameter_types! {
 	pub const KusamaForStatemint: (MultiAsset, MultiLocation) =
-		(AllConcreteFungible { id: Null }, X1(Parachain(1000)));
+		(AllConcreteFungible { id: MultiLocation::empty() }, MultiLocation::with_parachain_interior(1000));
```

Comment:
I find this to be confusing. `with_parachain_interior` means 1 child, which is a parachain, with this id? what would be the raw equivalent of this?

Reply:
It will just be this:
```rust
MultiLocation {
    parents: 0,
    interior: X1(Parachain(1000)),
}
```

Since I've privatized the fields on `MultiLocation`, it isn't possible to create such a struct outside of the `xcm` crate. Moreover, we're currently in a const eval context, which means we can only call const functions and thereby leading me to create a new `with_parachain_interior` const function on `MultiLocation` to allow construction of this kind of `MultiLocation`.

Reply:
This whole const eval context is self-imposed, you don't need to make `parameter_types` const. Just remove the keyword `const` from there and it will not require it to be const anymore.